airflow/utils/task_group.py

```diff
@@ -348,10 +382,35 @@ def build_map(task_group):
         build_map(self)
         return task_group_map
 
-    def get_child_by_label(self, label: str) -> Union["BaseOperator", "TaskGroup"]:
+    def get_child_by_label(self, label: str) -> DAGNode:
         """Get a child task/TaskGroup by its label (i.e. task_id/group_id without the group_id prefix)"""
         return self.children[self.child_id(label)]
 
+    def map(self, arg: Iterable) -> "MappedTaskGroup":
+        if self.children:
+            raise RuntimeError("Cannot map a TaskGroup that already has children")
+        if not self.group_id:
+            raise RuntimeError("Cannot map a TaskGroup before it has a group_id")
+        if self._parent_group:
+            self._parent_group._remove(self)
+        tg = MappedTaskGroup(self._group_id)
+        tg.mapped_arg = arg
+        tg.mapped_kwargs = {}
+        tg.partial_kwargs = {}
```

Comment:
Any reason these aren’t in `MappedTaskGroup`’s `__init__`?

Reply:
Nope, no idea why they weren't - just fixed.